app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -210,6 +210,9 @@ public void onReceive(Context context, Intent intent) {
                 setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
             }
         }
+        else {
+            setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_USER);
+        }
```

Comment:
I don't think so,since it's part of the onCreate() and only executed on restart.

The only reason I put it here is I was under the impression that setRequestedOrientation() sometimes persists between restarts :\